scripts/bench.py

```diff
@@ -56,13 +57,20 @@ def readfile(filepath, is_symmetric, backend):
             return ga.Graph(A)
         return ga.DiGraph(A)
     a = scipy.io.mmread(filepath)
-    if backend == "networkx":
+    if backend in {"networkx", "igraph"}:
         create_using = nx.Graph if is_symmetric else nx.DiGraph
-        return nx.from_scipy_sparse_array(a, create_using=create_using)
+        G = nx.from_scipy_sparse_array(a, create_using=create_using)
+        if backend == "networkx":
+            return G
+        if backend == "igraph":
+            # TODO: is there a better way for igraph to read MM files or scipy.sparse arrays?
```

Comment:
You can use `igraph.Graph.Adjacency` to convert from a SciPy sparse matrix.

There is no direct support for reading MM files in igraph.